### ISSUE TITLE
UI, Spelling, Alignments, Grammar Issues

### ISSUE THREAD

**Issue by zetaleona** [vasavah/vsv-fli-pass-uat#26]
**Describe the Bug**
•	Invitation notice: “You already sent an Invitation, Would you like to send an Email again?.”
o	Should be “This vendor has already been invited. Would you like to send an email again?”

![ss](https://user-images.githubusercontent.com/50394125/61112335-30254080-a4be-11e9-86dd-00a53590100d.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Fields: “Unit No./floor”, “Street”, “Subdivision/Village”, and “Building”

**Expected Behavior**
o	Change: Make fields NOT REQUIRED 

![2](https://user-images.githubusercontent.com/50394125/61112725-189a8780-a4bf-11e9-94cb-0203c861763e.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Field Label from “Executive Distributor”
o	Should be “Exclusive Distributor”
o	Change: Field type from ‘free text’ to ‘dropdown list’ with values of “Yes” and “No”

•	Field: “Authorized Distributor”
o	Change to “Brands/Products Authorized to Distribute”
o	Will only appear if “Exclusive Distributor” is answered with “Yes”

![3](https://user-images.githubusercontent.com/50394125/61112900-834bc300-a4bf-11e9-8ae0-3a968b5a8598.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Field: Country
o	Ampersand “&” appears as “&amp;”.
o	The USA is not part of the list.
![4](https://user-images.githubusercontent.com/50394125/61112979-afffda80-a4bf-11e9-9bbb-7cd6a10a466b.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Field: Industry
o	Field type should be a dropdown list
o	“Trade” should be a different field (this is part of the signed additional requirements from CPCAD)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Field: Company Website:
o	The system requires the user to include “http://” in order to proceed with the next step. This shouldn’t be the case.

**Comment by zetaleona** [comment]
**Describe the bug**
•	 ’Attach File’ from ‘Company’ tab
o	Change: Field Label from “Attach File” to “Attach Company Profile/Brochures/Catalogs/etc”

![5](https://user-images.githubusercontent.com/50394125/61113222-49c78780-a4c0-11e9-8207-cd2664ef9ab6.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Field: “Tax Identification Number (if applicable)” 
o	Change to “Taxpayer Identification Number (TIN)”
o	This field shouldn’t accept “e”, “.”, and “+”
![6](https://user-images.githubusercontent.com/50394125/61113258-65cb2900-a4c0-11e9-9058-eb1a10dda04e.png)

**Comment by zetaleona** [comment]
**Describe the bug**
o	Change: Field that while typing, there is a separator (-) for every three digits (except after the 3rd separator). Maximum characters should be 14 characters i.e. XXX-XXX-XXX-XXXXX (not including separator)

o	If possible, fill in the last 5 digits automatically if only 9 characters are inputted i.e. XXX-XXX-XXX-00000; If not possible, Minimum no. of characters should be 9 characters.

o	Change: Field – will only appear if the vendor chose “local” in “classification” field, make this field REQUIRED

![7](https://user-images.githubusercontent.com/50394125/61113346-ac208800-a4c0-11e9-990b-7b2a006ef70a.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Field: “Tax Payer Classification”
•	Change: Field type from ‘free text’ to ‘dropdown list’ with values of “VAT
 •	Registered” and “Non-VAT Registered”
![8](https://user-images.githubusercontent.com/50394125/61113422-d96d3600-a4c0-11e9-93cf-50c1635138e5.png)

**Comment by zetaleona** [comment]
**Describe the bug**
Change: Label: “Contact Person” to “Contact Persons”
•	Make more prominent i.e. all caps, bigger font, etc.
•	Add the line below label (separator)
Change: Label: “Executive” to “Owner”

![9](https://user-images.githubusercontent.com/50394125/61113597-37018280-a4c1-11e9-84da-c833eed126f2.png)

**Comment by zetaleona** [comment]
**Describe the bug**
FINANCIAL INFORMATION PAGE:
•	Years in Net Profit-Expenses Comparison should be in order
![10](https://user-images.githubusercontent.com/50394125/61113693-5f897c80-a4c1-11e9-897b-45b6c977cef1.png)
![11](https://user-images.githubusercontent.com/50394125/61113748-73cd7980-a4c1-11e9-8143-eb61b9b85f2b.png)

**Comment by zetaleona** [comment]
**Describe the bug**
Account Type: [dropdown: Current/Savings] 
Change: “Attach Bank Account Certificate” to “Attach Bank Information Sheet”


![12](https://user-images.githubusercontent.com/50394125/61113813-a4adae80-a4c1-11e9-9346-9a0349e77dc5.png)

**Comment by zetaleona** [comment]
**Describe the bug**
Transfer “Statement of Income”, “Statement of Financial Position”, and “Statement of Cashflow” from “Government” to “Financial” tab
Change: label: from “Additional Requirements” to “Financial Requirements”
NOT REQUIRED
![13](https://user-images.githubusercontent.com/50394125/61113994-fce4b080-a4c1-11e9-8cb2-d29a58740dfd.png)

**Comment by zetaleona** [comment]
**Describe the bug**
Change: Label: to “PCAB Contractor’s License”
Make NOT REQUIRED. This is a Contractor requirement.

Change: Label: “DOLE Certification of No Pending Case”
Make NOT REQUIRED. This is a Contractor requirement.

Change: label: to For Corporation/Partnership

Change: label: to Articles of Incorporation/Partnership

Latest General Information Sheet (GIS) and Secretary’s Certificate should be transferred to “For Corporation”
Attach Valid ID (Gov’t Issued) and TIN ID should be transferred to “For Sole Proprietorship”
Change: Label: to ”Attach Valid ID (Gov’t Issued) and TIN ID of the owner”

![14](https://user-images.githubusercontent.com/50394125/61114173-5cdb5700-a4c2-11e9-9e76-e28ab1104fa3.png)

**Comment by zetaleona** [comment]
**Describe the bug**
BUSINESS LEGITIMACY:
•	Please remove instructions under Business Legitimacy as this is not applicable anymore.

![15](https://user-images.githubusercontent.com/50394125/61114354-be032a80-a4c2-11e9-8caf-34557ae8c470.png)

**Comment by zetaleona** [comment]
**Describe the bug**
•	Active Suppliers *remove*
•	Accredited Active Suppliers change to Accredited Vendors
•	Completion change to Vendor Application Completion
o	Remove decimals. This should be rounded off to the nearest tens digit

•	**Total No. of Suppliers change to Total No. of Vendors**
o	add: Breakdown
o	Accredited:
o	Invited:
o	Ongoing accreditation: 
o	For review and approval:
![16](https://user-images.githubusercontent.com/50394125/61114629-513c6000-a4c3-11e9-969c-a10d9e4686a8.png)

**Comment by zetaleona** [comment]
Change: icon for Accredited Vendors Card (dashboard)

![17](https://user-images.githubusercontent.com/50394125/61114834-bee88c00-a4c3-11e9-8b8b-3d93f3e7e1a3.png)

**Comment by zetaleona** [comment]
Change: icon for Vendor Application Completion Card (dashboard)

![18](https://user-images.githubusercontent.com/50394125/61114955-fe16dd00-a4c3-11e9-8743-7c84cba28438.png)

**Comment by zetaleona** [comment]
•	“Rejected and Suspended” should be “Rejected”.
•	Should be “Expired Accreditation”

![19](https://user-images.githubusercontent.com/50394125/61115116-56e67580-a4c4-11e9-85e5-d6fa2b9786a5.png)

**Comment by zetaleona** [comment]
•	This graph should follow the color scheme of the 2nd graph

![21](https://user-images.githubusercontent.com/50394125/61115244-a0cf5b80-a4c4-11e9-99bb-1e168d89975e.png)

![22](https://user-images.githubusercontent.com/50394125/61115251-a331b580-a4c4-11e9-9f0b-eec2113f64c0.png)

**Comment by zetaleona** [comment]
o Please change the background color of “3” “Fair” so that it won’t blend in with the other parts of the Credentials tab.

![23](https://user-images.githubusercontent.com/50394125/61115371-e68c2400-a4c4-11e9-8e35-a949cee2d4d5.png)

**Comment by HannahCastro** [comment]
> **Describe the bug**
> Change: Label: “Contact Person” to “Contact Persons”
> •	Make more prominent i.e. all caps, bigger font, etc.
> •	Add the line below label (separator)
> Change: Label: “Executive” to “Owner”
> 
> ![9](https://user-images.githubusercontent.com/50394125/61113597-37018280-a4c1-11e9-84da-c833eed126f2.png)

Please note that this error (change label from "Executive" to "Owner") is part of the signed additional requirements for PASS.

**Comment by HannahCastro** [comment]
> **Describe the bug**
> •	Field: “Tax Payer Classification”
> •	Change: Field type from ‘free text’ to ‘dropdown list’ with values of “VAT
> •	Registered” and “Non-VAT Registered”
> ![8](https://user-images.githubusercontent.com/50394125/61113422-d96d3600-a4c0-11e9-93cf-50c1635138e5.png)

Please note that this error is part of the signed additional requirement from CPCAD (VAT Information - VAT or Non-VAT)

**Comment by zetaleona** [comment]
> **Describe the bug**
> •	Field: Industry
> o	Field type should be a dropdown list
> o	“Trade” should be a different field (this is part of the signed additional requirements from CPCAD)

**Comment by Lex-Castro** [comment]
> Describe the bug
> Account Type: [dropdown: Current/Savings]
> Change: “Attach Bank Account Certificate” to “Attach Bank Information Sheet”

Fields relating to EastWest Bank Details are still wrong (issue was assigned a ticket but details were incomplete in ticket; complete details are in **Consolidated adjustments and errors** that was closed)

**CURRENT:** 

- EastWest Bank Branch Name is asked for every year

- Account number is asked for every year

- EastWest Account Details are asked for both local and international vendors

**EXPECTED RESULT:**

- EastWest Bank Branch Name, Account Name, Account Number, Account type (dropdown: Current/Savings) should be asked only once (not for each year)

- EastWest Account Details are only asked for local vendor

- There should be a line separating the data between years

**IN ADDITION**

- when viewing **Financial Capacity**, the year of the financial data are not indicated (compared to when editing/filling out as vendor, year is indicated)




![image](https://user-images.githubusercontent.com/52643156/61339598-0e86d900-a871-11e9-9d69-a42bf562c858.png)
![image](https://user-images.githubusercontent.com/52643156/61339787-fbc0d400-a871-11e9-900c-817adfede451.png)
![image](https://user-images.githubusercontent.com/52643156/61339862-5f4b0180-a872-11e9-94ca-3ca6b5506035.png)

**Comment by HannahCastro** [comment]
> **Describe the bug**
> •	Field: Country
> o	Ampersand “&” appears as “&”.
> o	The USA is not part of the list.
> ![4](https://user-images.githubusercontent.com/50394125/61112979-afffda80-a4bf-11e9-9bbb-7cd6a10a466b.png)

Done.

**Comment by HannahCastro** [comment]
> •	“Rejected and Suspended” should be “Rejected”.
> •	Should be “Expired Accreditation”
> 
> ![19](https://user-images.githubusercontent.com/50394125/61115116-56e67580-a4c4-11e9-85e5-d6fa2b9786a5.png)

Dashboard has been edited. However, if you click the tabs, the page header doesn't match the tab name.

![image](https://user-images.githubusercontent.com/51902789/61357793-8b36a900-a8ab-11e9-8c4d-15cbfca0fbcc.png)
![image](https://user-images.githubusercontent.com/51902789/61357891-ba4d1a80-a8ab-11e9-9174-294839bf1513.png)

**As of August 13, this is now okay.**

**Comment by HannahCastro** [comment]
> **Describe the Bug**
> •	Invitation notice: “You already sent an Invitation, Would you like to send an Email again?.”
> o	Should be “This vendor has already been invited. Would you like to send an email again?”
> 
> ![ss](https://user-images.githubusercontent.com/50394125/61112335-30254080-a4be-11e9-86dd-00a53590100d.png)

Done.

**Comment by Lex-Castro** [comment]
> Describe the bug
> Change: Label: to “PCAB Contractor’s License”
> Make NOT REQUIRED. This is a Contractor requirement.
> Change: Label: “DOLE Certification of No Pending Case”
> Make NOT REQUIRED. This is a Contractor requirement.
> Change: label: to For Corporation/Partnership
> Change: label: to Articles of Incorporation/Partnership
> Latest General Information Sheet (GIS) and Secretary’s Certificate should be transferred to “For Corporation”
> Attach Valid ID (Gov’t Issued) and TIN ID should be transferred to “For Sole Proprietorship”
> Change: Label: to ”Attach Valid ID (Gov’t Issued) and TIN ID of the owner”

**ADD:**
Field: "Type of Business Organization"; Dropdown (choices: "Sole Proprietor" & 'Partnership/Coorporation")
Required uploads "For corporation/partnership" and "for Sole proprietor" are hidden(not required) and visible (required) accordingly

_Previous request was to add labels and required uploads, but the issue now is that all of what was added are "required" and this will be impossible for the vendor to comply_

**Comment by HannahCastro** [comment]
> o Please change the background color of “3” “Fair” so that it won’t blend in with the other parts of the Credentials tab.
> 
> ![23](https://user-images.githubusercontent.com/50394125/61115371-e68c2400-a4c4-11e9-8e35-a949cee2d4d5.png)

Done.

**Comment by Lex-Castro** [comment]
> Describe the bug
> •	Fields: “Unit No./floor”, “Street”, “Subdivision/Village”, and “Building”
> Expected Behavior
> o	Change: Make fields NOT REQUIRED

Hi,

As a vendor, when filling out  above mentioned fields, the fields are shown w/o asterisk (not required) but system still requires the said fields. (it's as if the fields are still required)

**Comment by HannahCastro** [comment]
> **Describe the bug**
> •	Field: Industry
> o	Field type should be a dropdown list
> o	“Trade” should be a different field (this is part of the signed additional requirements from CPCAD)

Industry is now a dropdownlist but Trade is still not included.

**Comment by HannahCastro** [comment]
> **Describe the bug**
> •	Field: “Tax Identification Number (if applicable)”
> o	Change to “Taxpayer Identification Number (TIN)”
> o	This field shouldn’t accept “e”, “.”, and “+”
> ![6](https://user-images.githubusercontent.com/50394125/61113258-65cb2900-a4c0-11e9-9058-eb1a10dda04e.png)

TIN is no longer tagged as required "*" but the system still requires it even if the vendor is international.
![image](https://user-images.githubusercontent.com/51902789/61605151-c7d11e80-ac76-11e9-9c66-13ba06072cd5.png)

**Comment by Lex-Castro** [comment]
> •	This graph should follow the color scheme of the 2nd graph
Done. now resolved.

**Comment by Lex-Castro** [comment]
> •	This graph should follow the color scheme of the 2nd graph

Done

**Comment by paulinapaulinapaulina** [comment]
Please address new comments.

**Comment by Lex-Castro** [comment]
> Describe the bug
> Account Type: [dropdown: Current/Savings]
> Change: “Attach Bank Account Certificate” to “Attach Bank Information Sheet”
> 
> Fields relating to EastWest Bank Details are still wrong (issue was assigned a ticket but details were incomplete in ticket; complete details are in Consolidated adjustments and errors that was closed)
> CURRENT:
> 
> 
> EastWest Bank Branch Name is asked for every year
> 
> 
> Account number is asked for every year
> 
> 
> EastWest Account Details are asked for both local and international vendors
> 
> 
> EXPECTED RESULT:
> 
> 
> EastWest Bank Branch Name, Account Name, Account Number, Account type (dropdown: Current/Savings) should be asked only once (not for each year)
> 
> 
> EastWest Account Details are only asked for local vendor
> 
> 
> There should be a line separating the data between years
> 
> 
> IN ADDITION
> 
> when viewing Financial Capacity, the year of the financial data are not indicated (compared to when editing/filling out as vendor, year is indicated)

These are still not reflected in PASS. EastWest Bank Account is still asked for every year. Will create a separate ticket for this.

**Comment by HannahCastro** [comment]
> > Describe the bug
> > Change: Label: to “PCAB Contractor’s License”
> > Make NOT REQUIRED. This is a Contractor requirement.
> > Change: Label: “DOLE Certification of No Pending Case”
> > Make NOT REQUIRED. This is a Contractor requirement.
> > Change: label: to For Corporation/Partnership
> > Change: label: to Articles of Incorporation/Partnership
> > Latest General Information Sheet (GIS) and Secretary’s Certificate should be transferred to “For Corporation”
> > Attach Valid ID (Gov’t Issued) and TIN ID should be transferred to “For Sole Proprietorship”
> > Change: Label: to ”Attach Valid ID (Gov’t Issued) and TIN ID of the owner”
> 
> **ADD:**
> Field: "Type of Business Organization"; Dropdown (choices: "Sole Proprietor" & 'Partnership/Coorporation")
> Required uploads "For corporation/partnership" and "for Sole proprietor" are hidden(not required) and visible (required) accordingly
> 
> _Previous request was to add labels and required uploads, but the issue now is that all of what was added are "required" and this will be impossible for the vendor to comply_

This is now done.
